modules/oe_whitelabel_search/README.md

```diff
@@ -0,0 +1,119 @@
+# OpenEuropa Whitelabel Search
+
+This module offers a searchbox block handled by a plugin.
+
+# Requirements
+
+The following modules must be installed and enabled to use the Search Box block:
+ - [Search_api](https://www.drupal.org/project/search_api)
+ - [Search_api_autocomplete](https://www.drupal.org/project/search_api_autocomplete)
+ - [Views](https://www.drupal.org/docs/8/core/modules/views)
+ - [Views UI](https://www.drupal.org/docs/8/core/modules/views-ui)
+
+# Instructions:
+
+The following instructions are meant to help site builders place a block Search Box in the OE Whitelabel theme.
+
+ - Enable the module OpenEuropa Whitelabel Search (oe_whitelabel_search).
+ - Go to section [Place the Search Box block](#place-the-search-box-block) if only the block is needed.
+ - If content type is ready to be used, skip section `Create a content type Test`. For the instructions on the Search API and Views, go to next sections.
+
+## Create a content type Test
```

Comment:
all this is not needed, just explain what needs to be added in composer if anything